assets/js/frontend/give-checkout-global.js

```diff
@@ -63,8 +63,16 @@ jQuery(function ($) {
 					// Check if user want to show the feilds or not.
 					if (typeof ( response.show_field ) != undefined && true == response.show_field) {
 						$form.find('p#give-card-state-wrap').removeClass('give-hidden');
+
```

Comment:
This file was being remove soo add the new code in the below commit 
https://github.com/WordImpress/Give/pull/2284/commits/0e17396b2cc94d60b44e82088f5f3c8391487902